### DISCUSSION TITLE
Res101 vs Midas?

### DISCUSSION THREAD

**Discussion by ice051128** [thygate/stable-diffusion-webui-depthmap-script#52] [Q&A] [answered]
Is it normal for res101 to destroy all midas models? It's much faster while being more accurate

**Comment by thygate** [comment, marked as answer]
Yes, I was surprised by this too .. I'm not sure about accuracy though .. I've had some res101 results where the depthmap was somewhat inaccurate .. The midas models aren't perfect either of course, but for the moment res101 is my preference too. It also uses a lot less VRAM than the dpt_* ones ..
